### PR TITLE
PMM-7 merge

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-7

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: .github/workflows/ci-api.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    needs: [check]
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Run Merge Gatekeeper
+        uses: percona-platform/merge-gatekeeper@v1.0.2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          interval: 45
+          timeout: 1200
+          ref: ${{ github.sha }}
```
Comment (artemgavrilov) [inline .github/workflows/ci-api.yml:110]: I guess we don't need it here.

File: .github/workflows/ci-api.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
+name: CI
```
Comment (artemgavrilov) [inline .github/workflows/ci-api.yml:1]: `API CI` or something like this

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -112,6 +89,7 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           interval: 45
           timeout: 1200
+          ref: ${{ github.event.pull_request.head.sha || github.event.push.head.sha }}
```
Comment (artemgavrilov) [inline .github/workflows/ci.yml:92]: I wonder what is the purpose of this option? 🤔

Comment (BupycHuk) [reply .github/workflows/ci.yml:92]: merge gatekeeper is failing in main branch

Comment (BupycHuk) [reply .github/workflows/ci.yml:92]: https://github.com/percona/pmm/runs/6908254035?check_suite_focus=true

File: .github/workflows/ci-api.yml (hunk 1 above)
Comment (BupycHuk) [reply .github/workflows/ci-api.yml:110]: You are right 👍

**Review by idoqo — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -66,31 +64,9 @@ jobs:
...
-          make descriptors
-          git diff --text
-
       - name: Check build
         run: make release
 
-      - name: Run linters
-        env:
-          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: make ci-reviewdog
```
Comment (artemgavrilov) [inline .github/workflows/ci.yml:92]: @BupycHuk it seems that this part was missed

Comment (BupycHuk) [reply .github/workflows/ci.yml:92]: oh, shit

**Pull request merged**